### ISSUE TITLE
Definitions that link to WCAG Understanding docs are either going to wrong version of WCAG (2.1) or have a broken link

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag#3184]
- ** Links going to WCAG 2.1** [accessibility supported](https://www.w3.org/TR/WCAG22/#dfn-accessibility-supported) Note 1 should be linking to [Level of Assistive Technology Support Needed for "Accessibility Support"](https://www.w3.org/WAI/WCAG22/Understanding/conformance#support-level) and last note should be linking to [Understanding Accessibility-Supported Web Technology Uses](https://www.w3.org/WAI/WCAG22/Understanding/conformance#documented-lists) https://www.w3.org/WAI/WCAG21/Understanding/conformance#documented-lists
- * Link going to WCAG 2.1** [conforming alternate version](https://www.w3.org/TR/WCAG22/#dfn-conforming-alternate-versions) should be linking to https://www.w3.org/WAI/WCAG22/Understanding/conformance#conforming-alt-versions for the Understanding Conforming Alternate Versions.
- **link broken** [relative luminance](https://www.w3.org/TR/WCAG22/#dfn-relative-luminance) should be linking to: https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html for the Understanding Success Criterion 1.4.3 link in the first note.
- * Link going to WCAG 2.1** [text alternative](https://www.w3.org/TR/WCAG22/#dfn-text-alternative) should be linking to [Understanding Text Alternatives](https://www.w3.org/WAI/WCAG22/Understanding/conformance#text-alternatives) at https://www.w3.org/WAI/WCAG22/Understanding/conformance#text-alternatives

Not sure if the appropriate links are supposed to be inserted through scripting or are fully defined in the definitions, but these need fixing.

**Comment by alastc** [comment]
Hi @maryjom, 

Thanks for finding where they should go to, but I'm struggling to locate where they are linked from. Doing a search in the codebase for those doesn't seem to work, so I'm not sure where they link *from*.

**Comment by maryjom** [comment]
@alastc 
- accessibility supported link is in this file: https://github.com/w3c/wcag/blob/main/guidelines/terms/20/accessibility-supported.html
- conforming alternate version link is in this file: https://github.com/w3c/wcag/blob/main/guidelines/terms/20/conforming-alternate-version.html
- relative luminance broken link is in this file: https://github.com/w3c/wcag/blob/main/guidelines/terms/20/relative-luminance.html
- Text alternative link is in this file: https://github.com/w3c/wcag/blob/main/guidelines/terms/20/text-alternative.html

**Comment by maryjom** [comment]
Seems that if only link targets change and none of the normative text changes, that the definition file is kept in the directory of the version of WCAG that introduced the term.

**Comment by maryjom** [comment]
This is still a problem. Since WCAG2ICT is embedding the Understanding Intent and WCAG 2 source code for definitions, the links in these files are either going to WCAG 2.1 or broken. On today's WCAG2ICT call, @bruce-usab mentioned this and opened [WCAG2ICT Issue 294](https://github.com/w3c/wcag2ict/issues/294). I have a PR for the one issue that was in the WCAG2ICT source, but the rest are in WCAG's source code.

**Comment by kfranqueiro** [comment]
These have been resolved in #3438 and #4586. Changes from the latter PR are only currently reflected in the [editor's draft](https://w3c.github.io/wcag/guidelines/22/).
